dali_mqtt_daemon.py

```diff
@@ -218,13 +233,22 @@ def on_connect(
                 actual_level.value,
                 retain=True,
             )
+
             client.publish(
                 MQTT_STATE_TOPIC.format(mqqt_base_topic, lamp),
                 MQTT_PAYLOAD_ON if actual_level.value > 0 else MQTT_PAYLOAD_OFF,
                 retain=True,
             )
+            lamps_summary_list.append(
+                {
+                    "short_address": short_address.address,
+                    "brightness": actual_level.value,
```

Comment:
I don't see sens in adding third variable name aka brightness. It should be "actual_level": actual_level.value,

Reply:
You look at outdated code